app/models/page.rb

```diff
@@ -35,6 +35,14 @@ class Page < ApplicationRecord
 
   default_scope { order(title: :asc) }
 
+  def get_all_categories
```

Comment:
This should be a class method, `self.all_categories`. It's not an enforced rule, but we typically avoid `get_` and `set_` functions unless necessary.

Reply:
Yes, this method meant for the whole class (like a static method or class method) rather than meant for a single instance/object.
I thought the way I did it does make it a class method already (coz I am not using `self` in the method body). Thanks for this